properties-panel/README.md

```diff
@@ -7,47 +7,103 @@ This example uses [bpmn-js](https://github.com/bpmn-io/bpmn-js) and [bpmn-js-pro
 
 This example is a node-style web application that builds a user interface around the bpmn-js BPMN 2.0 modeler.
 
-![demo application screenshot](https://raw.githubusercontent.com/bpmn-io/bpmn-js-examples/master/properties-panel/docs/screenshot.png "Screenshot of the modeler + properties panel example")
+![demo application screenshot](./docs/screenshot.png "Screenshot of the modeler + properties panel example")
```

Comment:
I'd give the properties panel in the screenshot slightly more width + use a less prominent example (i.e. without colors).

It does make sense to use a real-world example though.

Reply:
If you have a diagram I could use, feel free to share it! :)

Reply:
I used the pizza collaboration and increased the panel width to 400px. @nikku please check now.

Reply:
Great. If you move the diagram in sight (avoid participant-screen border collapsing) and focus an activity with less noise around it you'll end up with the perfect screenshot.

Reply:
Done.